x-pack/test/reporting/functional/reporting.js

```diff
@@ -27,275 +23,141 @@ export default function({ getService, getPageObjects }) {
-          this.timeout(360000);
-
-          await PageObjects.dashboard.switchToEditMode();
-          await PageObjects.dashboard.useMargins(true);
-          await PageObjects.dashboard.saveDashboard('report test');
-          await PageObjects.reporting.openPdfReportingPanel();
-          await PageObjects.reporting.checkUsePrintLayout();
-          await PageObjects.reporting.clickGenerateReportButton();
-          await PageObjects.reporting.clickDownloadReportButton(60000);
-          PageObjects.reporting.clearToastNotifications();
-
-          const url = await PageObjects.reporting.getUrlOfTab(1);
-          const reportData = await PageObjects.reporting.getRawPdfReportData(url);
-
-          await PageObjects.reporting.closeTab(1);
-          const reportFileName = 'dashboard_print';
-          const sessionReportPath = await writeSessionReport(reportFileName, reportData);
-          const percentSimilar = await checkIfPdfsMatch(
-            sessionReportPath,
-            getBaselineReportPath(reportFileName),
-            config.get('screenshots.directory'),
-            log
-          );
-          // After expected OS differences, the diff count came to be around 128k
-          expect(percentSimilar).to.be.lessThan(0.05);
-        });
-      });
-
-      // TODO Re-enable the tests after removing Phantom:
-      // https://github.com/elastic/kibana/issues/21485
-      describe.skip('Preserve Layout', () => {
-        it('matches baseline report', async function() {
-          // Generating and then comparing reports can take longer than the default 60s timeout because the comparePngs
-          // function is taking about 15 seconds per comparison in jenkins.
-          this.timeout(360000);
-
-          await PageObjects.reporting.openPdfReportingPanel();
-          await PageObjects.reporting.forceSharedItemsContainerSize({ width: 1405 });
-          await PageObjects.reporting.clickGenerateReportButton();
-          await PageObjects.reporting.removeForceSharedItemsContainerSize();
-
-          await PageObjects.reporting.clickDownloadReportButton(60000);
-          PageObjects.reporting.clearToastNotifications();
-
-          const url = await PageObjects.reporting.getUrlOfTab(1);
-          await PageObjects.reporting.closeTab(1);
-          const reportData = await PageObjects.reporting.getRawPdfReportData(url);
-
-          const reportFileName = 'dashboard_preserve_layout';
-          const sessionReportPath = await writeSessionReport(reportFileName, reportData);
 
-          const percentSimilar = await checkIfPdfsMatch(
-            sessionReportPath,
-            getBaselineReportPath(reportFileName),
-            config.get('screenshots.directory'),
-            log
-          );
-          expect(percentSimilar).to.be.lessThan(0.05);
+          const url = await PageObjects.reporting.getReportURL(60000);
+          expect(url).to.match(/download/);
```

Comment:
@joelgriffith I found that I was not able to use the same code here as there is in master, where we're actually sending a request out for the URL and checking response headers for OK values. In this branch, the `url` value returned is relative, so the fetch was failing since it didn't have the correct host or port number. 

There aren't very many code differences between 7.x and master in this branch, so I'm not sure what the difference is that is making the return URL something that can't be requested. That's why this just does a regex match on the returned URL string.